components/application-broker/internal/broker/deprovision_test.go

```diff
@@ -234,7 +234,7 @@ func TestDoDeprovision(t *testing.T) {
 				fixDeprovisionSucceeded(),
 			).Return(nil).Once()
 
-			knCli, k8sCli := bt.NewFakeClients(tc.initialObjs...)
+			knCli, k8sCli, _ := bt.NewFakeClients(tc.initialObjs...)
```

Comment:
I know in deprovisioning process policy removal is not needed. It is based on the decision that we will keep the broker after the deprovisioning process along with the policy. I think a comment on the background with a Github issue link will be helpful so that in future we know why `istioclientset` is ignored here.